packages/x-date-pickers/src/DesktopNextTimePicker/DesktopNextTimePicker.tsx

```diff
@@ -0,0 +1,280 @@
+   */
+  disableOpenPicker: PropTypes.bool,
+  /**
+   * If `true` disable values after the current time.
+   * @default false
+   */
+  disablePast: PropTypes.bool,
+  /**
+   * Format of the date when rendered in the input(s).
+   * Defaults to localized format based on the used `views`.
+   */
+  format: PropTypes.string,
+  /**
+   * Pass a ref to the `input` element.
+   */
+  inputRef: PropTypes.oneOfType([
+    PropTypes.func,
+    PropTypes.shape({
+      current: PropTypes.object,
+    }),
+  ]),
+  /**
+   * The label content.
+   */
+  label: PropTypes.node,
+  /**
+   * Locale for components texts.
+   * Allows overriding texts coming from `LocalizationProvider` and `theme`.
+   */
+  localeText: PropTypes.object,
+  /**
+   * Max time acceptable time.
+   * For input validation date part of passed object will be ignored if `disableIgnoringDatePartForTimeValidation` not specified.
+   */
+  maxTime: PropTypes.any,
+  /**
+   * Min time acceptable time.
+   * For input validation date part of passed object will be ignored if `disableIgnoringDatePartForTimeValidation` not specified.
+   */
+  minTime: PropTypes.any,
+  /**
+   * Step over minutes.
+   * @default 1
+   */
+  minutesStep: PropTypes.number,
+  /**
+   * Callback fired when date is accepted @DateIOType.
+   * @template TValue
+   * @param {TValue} value The value that was just accepted.
+   */
+  onAccept: PropTypes.func,
+  /**
+   * Callback fired when the value (the selected date) changes.
+   * @template TValue
+   * @param {TValue} value The new value.
+   */
+  onChange: PropTypes.func,
+  /**
+   * Callback fired when the popup requests to be closed.
+   * Use in controlled mode (see open).
+   */
+  onClose: PropTypes.func,
+  /**
+   * Callback that fired when input value or new `value` prop validation returns **new** validation error (or value is valid after error).
+   * In case of validation error detected `reason` prop return non-null value and `TextField` must be displayed in `error` state.
+   * This can be used to render appropriate form error.
+   *
+   * [Read the guide](https://next.material-ui-pickers.dev/guides/forms) about form integration and error displaying.
+   * @DateIOType
+   *
+   * @template TError, TValue
+   * @param {TError} reason The reason why the current value is not valid.
+   * @param {TValue} value The invalid value.
+   */
+  onError: PropTypes.func,
+  /**
+   * Callback fired when the popup requests to be opened.
+   * Use in controlled mode (see open).
+   */
+  onOpen: PropTypes.func,
+  /**
+   * Callback fired when the selected sections change.
+   * @param {FieldSelectedSections} newValue The new selected sections.
+   */
+  onSelectedSectionsChange: PropTypes.func,
+  /**
+   * Callback fired on view change.
+   * @template View
+   * @param {View} view The new view.
+   */
+  onViewChange: PropTypes.func,
+  /**
+   * Control the popup or dialog open state.
+   * @default false
+   */
+  open: PropTypes.bool,
```

Comment:
(not related to this PR)
I'm wondering if we should add some warning when `open` is `true` and the views as `null` to say "Hey, sounds you try to open the hour view without providing it, please ..."

Reply:
I agree that it could be a nice warning and very easy to implement.
If you want to have a look, you just have to look at `hasUIView` in `usePickerViews`